.travis.yml

```diff
@@ -21,6 +21,17 @@ after_success:
 
 jobs:
   include:
+    - stage: loopback.io
+      before_script:
+        - npm run bootstrap
+      script:
+        - cd loopback.io
+        - bundle install
+        - npm run fetch-readmes
```

Comment:
it knows to run `npm install` and `npm test` automatically in this stage right?

Reply:
See #1254 for the new PR. `npm test` is not run in the `loopback.io` (that's what the test stage is for). `install` happens as part of `bootstrap`.